src/agent/communicator/include/communicator.hpp

```diff
@@ -27,7 +27,15 @@ namespace communicator
             return m_tokenExpTime - now_seconds;
         }
 
+        std::string& GetManagerIp() { return m_managerIp; }
+        void SetManagerIp(const std::string_view ip) { m_managerIp = ip; }
```

Comment:
If we pass the configuration in the constructor and set these values in it, then there's no need to have setter methods.

(Also note that the methods were used in a previous commit, but only introduced here, at a later stage).

Reply:
Done.